### DISCUSSION TITLE
PC311-TY power clamp: Log warning

### DISCUSSION THREAD

**Discussion by MaximeNagel** [make-all/tuya-local#1045] [Q&A] [answered]
I have a warning message in log from HA regarding a power meter sensor  : PC311-TY power clamp

Is the warning normal, or maybe something wrong in the configuration yaml ? 

Entity sensor.consommation_global_maison_reactive_power_a (<class 'custom_components.tuya_local.sensor.TuyaLocalSensor'>) is using native unit of measurement 'kVar' which is not a valid unit for the device class ('reactive_power') it is using; expected one of ['var']; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.

**Comment by make-all** [comment, marked as answer]
Yes it was something wrong in the config.  For some reason the values were being scaled by 1000 to kVar, but they should just be passed in their original form as var.

**Reply by MaximeNagel** [reply]
Is this already fixed for the next version ?

Thx for your clear answer.

**Reply by make-all** [reply]
Yes, for 2023.9.0, which you can test now by selecting Redownload in HACS, and choosing "main" for the version to install. It is necessary to wait for the version select to re-enable after selecting "main" before pressing the Download button, otherwise it seems to download the default version (latest release) instead of what you selected.

**Reply by MaximeNagel** [reply]
Perfect ! Thanks :)
